src/components/ShoppingList.js

```diff
@@ -1,19 +1,36 @@
-import React from "react";
+import React, { useState } from "react";
 import Item from "./Item";
 
 function ShoppingList({ items }) {
+  const [selectedCategory, setSelectedCategory] = useState("All")
+  let newMap
+
+  function handleChange(event) {
+    setSelectedCategory(event.target.value)
+  }
+
+  newMap = []
+  items.filter(item => {
+    if (item.category === selectedCategory) {
+      newMap.push(item)
+    } else if (selectedCategory === "All") {
+      newMap = [...items]
+    }
+    return newMap
+  })
```

Comment:
You're using filter, which is great, but you're not REALLY using filter haha!
```suggestion
  const filterItems = items.filter((item) => {
    if(isSelected === 'All') {
      return items
    } else {
      return item.category === isSelected
    }
  })
```

Reply:
oopsie lol that makes sense